config/envs/default.py

```diff
@@ -72,6 +72,7 @@ class DefaultConfig(object):
     # Funding Service Design Post Award
     DISABLE_LOAD = "DISABLE_LOAD" in os.environ
     ENABLE_TF_R5 = os.getenv("ENABLE_TF_R5", True)
+    ENABLE_PF_R2 = os.getenv("ENABLE_PF_R2", True)
```

Comment:
I think we should default this to False? Or we need something like https://github.com/communitiesuk/funding-service-design-post-award-data-store/blob/1f3bc2e2caeb25fbb586b0685ae8103b200a9a0f/config/envs/default.py#L16 as any value will evaluate to truthy.